ItalianTranslation_PlanetCoaster.xml

```diff
@@ -2,93 +2,93 @@
+		<entry key="AdvancedTriggerEditor_MakeTriggerSequenceButton" translation="Crea Sequenza di Innesco"/>
+		<entry key="AdvancedTriggerEditor_PlaceTriggerOnTrack" translation="Posiziona un Innesco lungo il Percorso"/>
+		<entry key="AdvancedTriggerEditor_Play" translation="Testa l'attivazione sugli oggetti fissati"/>
+		<entry key="AdvancedTriggerEditor_PlayTriggerSequence" translation="Avvia Sequenza di Innesco"/>
 		<entry key="AdvancedTriggerEditor_PlayerTime" translation="{Minutes:int:minDigits=2}:{Seconds:int:minDigits=2}"/>
-		<entry key="AdvancedTriggerEditor_Racing_LapStepper" translation="Innesco solo su Giro"/>
-		<entry key="AdvancedTriggerEditor_Racing_PlaceStepper" translation="Innesco solo su Posizione"/>
-		<entry key="AdvancedTriggerEditor_Racing_TriggerAllLaps" translation="Innesco su tutti i Giri"/>
-		<entry key="AdvancedTriggerEditor_Racing_TriggerAllPlaces" translation="Innesco su tutte le Posizioni"/>
-		<entry key="AdvancedTriggerEditor_RemoveFromGroupToolTip" translation="Disconnetti da questo Gruppo Scenico"/>
-		<entry key="AdvancedTriggerEditor_RemoveFromTriggerToolTip" translation="Disconnetti da questo Innesco"/>
-		<entry key="AdvancedTriggerEditor_RemoveGroupToolTip" translation="Rimuovi questo gruppo"/>
-		<entry key="AdvancedTriggerEditor_RemoveTriggerToolTip" translation="Rimuovi questo Innesco"/>
-		<entry key="AdvancedTriggerEditor_ResetAllEventTimes" translation="Reset Ritardi Evento"/>
-		<entry key="AdvancedTriggerEditor_RideAddItems" translation="Collega oggetti a Innesco"/>
-		<entry key="AdvancedTriggerEditor_RideAddItemsToolTipDisabled" translation="Non puoi collegare altri oggetti a questo Innesco"/>
-		<entry key="AdvancedTriggerEditor_RideAddItemsToolTipEnabled" translation="Collega oggetti all'Innesco"/>
-		<entry key="AdvancedTriggerEditor_RideTriggersLabel" translation="Inneschi su Percorso"/>
-		<entry key="AdvancedTriggerEditor_ScenerySelectAvailableLabel" translation="Scenario Innescabile Disponibile"/>
-		<entry key="AdvancedTriggerEditor_ScenerySelectAvailableToolTip" translation="Scenario Innescabile Disponibile nel parco"/>
-		<entry key="AdvancedTriggerEditor_ScenerySelectDescription" translation="Usa lo strumento Multiselezione per selezionare e collgare Scenari Innescabili"/>
-		<entry key="AdvancedTriggerEditor_ScenerySelectGetMoreButton" translation="Ottieni più Scenari Innescabili"/>
-		<entry key="AdvancedTriggerEditor_ScenerySelectSelectedLabel" translation="Scenari Innescabili Selezionati"/>
-		<entry key="AdvancedTriggerEditor_ScenerySelectSelectedListLabel" translation="Scenari Innescabili"/>
-		<entry key="AdvancedTriggerEditor_ScenerySelectSelectedToolTip" translation="Scenari Innescabili Selezionati al momento"/>
+		<entry key="AdvancedTriggerEditor_Racing_LapStepper" translation="Attiva solo al Giro"/>
```

Comment:
Qui c'è da discutere su Innesco/Attivazione, mi sembrano giusti entrambi, però forse "Attiva" in questo caso è più comprensibile

Reply:
Approvo